config/external.js

```diff
@@ -0,0 +1,21 @@
+const pkgInfo = require('../package.json');
+
+let external = ['dns', 'fs', 'path', 'url'];
```

Comment:
Can we get code comments throughout all of this file to add more insight into (1) what we're doing and (2) why. Our target audience is your fellow maintainers -- folks experienced with at least webpack, but probably very unclear on what we're filtering on and why. Thanks!